### PR TITLE
Updated CPX, CIC images and RBAC

### PR DESCRIPTION
**Pull request by asparshk**

### CODE REVIEW

File: charts/examples/citrix-k8s-ingress-controller/templates/citrix-k8s-ingress.yaml [hunk 1]
```diff
@@ -4,17 +4,20 @@ metadata:
   name: cic-k8-role
 rules:
   - apiGroups: [""]
-    resources: ["services", "endpoints", "ingresses", "pods", "secrets"]
+    resources: ["services", "endpoints", "ingresses", "pods", "secrets", "routes", "routes/status"]
```
Comment (krdpk17) [inline charts/examples/citrix-k8s-ingress-controller/templates/citrix-k8s-ingress.yaml]: I don't see resource for namespaces and nodes. Kindly check

File: charts/stable/citrix-k8s-cpx-ingress-controller/templates/citrix-k8s-cpx-ingress.yaml [hunk 2]
```diff
@@ -7,14 +7,14 @@ rules:
 {{- if .Values.openshift }}
     resources: ["services", "endpoints", "ingresses", "pods", "secrets", "routes", "routes/status", "tokenreviews", "subjectaccessreviews"]
 {{- else }}
-    resources: ["services", "endpoints", "ingresses", "pods", "secrets"]
+    resources: ["services", "endpoints", "ingresses", "pods", "secrets", "routes", "routes/status"]
```
Comment (krdpk17) [inline charts/stable/citrix-k8s-cpx-ingress-controller/templates/citrix-k8s-cpx-ingress.yaml]: Same here. Please take care of at other places as well.

File: charts/examples/citrix-k8s-cpx-ingress-controller/templates/citrix-k8s-cpx-ingress.yaml [hunk 3]
```diff
@@ -4,17 +4,20 @@ metadata:
   name: cpx-ingress-k8-role
 rules:
   - apiGroups: [""]
-    resources: ["services", "endpoints", "ingresses", "pods", "secrets"]
+    resources: ["services", "endpoints", "ingresses", "pods", "secrets", "routes", "routes/status"]
```
Comment (krdpk17) [inline charts/examples/citrix-k8s-cpx-ingress-controller/templates/citrix-k8s-cpx-ingress.yaml]: Wished to check if we need routes/status here? Is it intentional change?

Comment (janraj) [reply charts/examples/citrix-k8s-cpx-ingress-controller/templates/citrix-k8s-cpx-ingress.yaml]: I think we should include  NS_NETPROFILE with yaml. What is your sugesstions @asparshk @krdpk17

Comment (krdpk17) [reply charts/examples/citrix-k8s-cpx-ingress-controller/templates/citrix-k8s-cpx-ingress.yaml]: @janraj yes. Thanks for mentioning this. We will discuss with you for the required change

**Comment by asparshk** [comment]
Addressed the review comments. @krdpk17 Please have a look.

**Pull request merged**